content/components/data-table.mdx

```diff
@@ -0,0 +1,598 @@
+
+</div>
+
+</Box>
+
+### Do's and Don'ts
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361204-99082cdf-36d7-4248-9a77-0bf58ee7fae5.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Use a data table for data in a flat hierarchical structure
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361206-e066a62d-40dd-4956-aa69-e2766c3b7d0b.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Don't use a data table for information that has hierarchical relationships
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361208-4191728f-b130-4c44-b834-da7dc29abc81.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Restrict the data in a column to represent one type of data
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361210-dcd62746-c10e-40dc-ba6d-7b6c509693e3.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Don't group data in a column cell that isn't described by the column header
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361212-b35c07de-4f66-4dfb-b2ac-bffd01723879.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Leave the cell blank {`(preferred)`} or show text that explains the absence
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361213-5bf83f35-ec56-4aa4-807b-3687fbef6d0a.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Don't show a character, icon, or emoji to indicate a cell is empty
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361215-c7f15b31-bf91-49ec-8ee0-f85af5c86588.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Allow the title in the page header to act as a table title when appropriate
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="456"
+      role="presentation"
+      src="https://user-images.githubusercontent.com/2313998/205361216-a10a8d67-5f04-4bf9-9b82-9ca851731698.png"
+    />
+    <Caption role="presentation" color="fg.subtle">
+      Don't repeat or rephrase the title in the page header as the table title
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+### Column widths
```

Comment:
Do we explicitly say anywhere that tables grow to fit 100% of the width of the container they are placed in?

Reply:
No, but do you think we need to?

Reply:
I think it'd be worth it, but I'm also paranoid about these sorts of things.